RevisionNotes/MIT_Lec_15.md

```diff
@@ -0,0 +1,29 @@
+# 15 - Projections into subspaces
+
+### The basics of projection
+![Projection image](./Images/MIT_15_projection.png)
+
+### Defining a basic formula for projecting b onto a line formed by a single vector.
+![Projection derivation example](./Images/MIT_15_basic.png)
+
+- Let `p` be the projection of `b` onto the line going through `a`.
+- Since `p` is at the shortest distance possible from `b`, the line joining them must me perpendicular to the line formed by multiples of `a`.
+- Therefore, if `e = b - p`, then e must be perpendicular to a.
+- Now `p` is a scalar multiple of `a`, so let `p = ax` with x being a scalar.
+- We get `a.t (b- ax) = 0`.
```

Comment:
Please use subspace notation in markdown instead of writing ".t".

Reply:
Made the changes